core/docs/spark-qualification-tool.md

```diff
@@ -49,6 +49,49 @@ Spark event logs after the application(s) have run, the second is to be integrat
 application using explicit API calls, and the third is to install a Spark listener which can output
 results on a per SQL query basis.
 
+In running the qualification tool standalone on Spark event logs, the tool can be run as a user tool command
+via a pip package for CSP environments (Google Dataproc, AWS EMR, Databricks AWS) or as a java application for
+other environments.
+
+## Running the Qualification tool standalone for CSP environments on Spark event logs
+### User Tools Prerequisites and Setup for CSP environments
+
+* [Dataproc](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-dataproc.md)
+* [EMR](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-aws-emr.md)
+* [Databricks AWS](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-databricks-aws.md)
+
+### Qualify CPU Workloads for GPU Acceleration
+
+The qualification tool will run against logs from your CSP environment and then will output the applications
+recommended for acceleration along with estimated speed-up and cost saving metrics.
```

Comment:
I think that "_cost saving metrics_" deserves more emphasis. It is one of the main advantages of running the pypi package. Maybe we can re-phrase one of the section headers to contain something like "How to use Qualification tool to get savings estimates on CSP"

Reply:
Good feedback.  Addressed in latest commit.